exec/java-exec/src/main/java/org/apache/drill/exec/server/rest/profile/ProfileResources.java

```diff
@@ -105,6 +109,25 @@ public String getTime() {
       return format.format(time);
     }
 
+    public long getStartTime() {
+      return startTime;
+    }
+
+    public long getEndTime() {
+      return endTime;
+    }
+
+    public String getDuration() {
```

Comment:
Done. This is a much simpler and cleaner solution. Thank you, @arina-ielchiieva !